package.json

```diff
@@ -148,6 +148,11 @@
                         "default": "- {problems_count} problems found",
                         "description": "The text to show in the status when there are problems. {problems_count} will be replaced with the number of problems."
                     },
+                    "vscord.status.problems.countedSeverities": {
```

Comment:
Maybe we can use an array of enum for this?

Reply:
Arrays arnt really a thing in vscode settings. Says u have to edit it in the settings.json file. With the enum you would also have 2 put every single combination if it did work